interface/admin.py

```diff
@@ -13,27 +13,20 @@
 
 @admin.register(Assignment)
 class AssignmentAdmin(admin.ModelAdmin):
-    actions = ['download_submissions']
-
-    def download_submissions(self, request, queryset):
-        if queryset.count() != 1:
-            messages.error(request, 'Only one assignment can be selected')
-            return
+    actions = ['download_review_submissions', 'download_all_submissions']
 
+    def get_submission_set(self, request, queryset):
```

Comment:
So this function simply returns `submission_set.order_by('timestamp')`? Why not remove it and move the code inline?